pilot/pkg/kube/inject/inject_test.go

```diff
@@ -473,6 +473,17 @@ func TestIntoResourceFile(t *testing.T) {
 			readinessPeriodSeconds:       DefaultReadinessPeriodSeconds,
 			readinessFailureThreshold:    DefaultReadinessFailureThreshold,
 		},
+		{
```

Comment:
I think the test for webhook was already merged?

Reply:
Yes, I have pick it here,  ready to close it but an owner has merged it yesterday...